src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -68,6 +70,10 @@ class OnfidoStep extends React.Component {
                     this.canShowOnfido() ? (
                         <Onfido
                             sdkToken={this.props.walletOnfidoData.sdkToken}
+                            onError={(error) => {
+                                Log.hmmm('Onfido error', {error});
```

Comment:
Maybe update these to include the step (`Onfido error in OnfidoStep`) since this can happen in multiple places